sim/mage/talents.go

```diff
@@ -403,8 +403,6 @@ func (mage *Mage) registerCombustionCD() {
 		Duration: time.Minute * 3,
 	}
 
-	fireCombCritMult := mage.SpellCritMultiplier(1, 0.1) / mage.SpellCritMultiplier(1, 0)
-
 	var fireSpells []*core.Spell
 	mage.OnSpellRegistered(func(spell *core.Spell) {
 		if spell.SpellSchool.Matches(core.SpellSchoolFire) && spell.Flags.Matches(SpellFlagMage) {
```

Comment:
❓ Mages still gain 10% crit chance per non-crit cast during combustion
<img width="326" alt="image" src="https://github.com/wowsims/sod/assets/12898988/27059d96-c52c-4786-9f6f-ead716adb849">

Reply:
Ye, but the "When activated, this spell increases your critical strike damage bonus with Fire damage spells by 50%..." part is gone, which is what this removes.

Reply:
Oooh right CritMultiplier that's crit damage. Ok cool lgtm